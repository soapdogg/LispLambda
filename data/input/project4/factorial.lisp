(DEFUN FACTORIAL (X)
    (COND
        ((LESS X 1) 1)
        (T (* X (FACTORIAL (- X 1))))
    )
)

(FACTORIAL 6)
(FACTORIAL 0)
(FACTORIAL 1)
(FACTORIAL (- 0 1))
(FACTORIAL 10)