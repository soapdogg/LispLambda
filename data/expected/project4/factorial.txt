720
1
1
1
3628800